cherrypy/test/test_httputil.py

```diff
@@ -30,3 +31,17 @@ def test_urljoin(self, script_name, path_info, expected_url):
         """Test all slash+atom combinations for SCRIPT_NAME and PATH_INFO."""
         actual_url = httputil.urljoin(script_name, path_info)
         assert actual_url == expected_url
+
+
+def test_valid_status():
+    code, reason, message = httputil.valid_status(200)
+    assert code == 200 and reason == 'OK'
+
+    code, reason, message = httputil.valid_status('500')
+    assert code == 500 and reason == 'Internal Server Error'
+
+    code, reason, message = httputil.valid_status(http_client.NOT_FOUND)
+    assert code == 404 and reason == 'Not Found'
+
+    with pytest.raises(ValueError):
```

Comment:
Could you please test more branches, so that coverage would not go down?

Reply:
The changed code has 100% coverage.  Overall coverage % can decrease when (covered) code is removed.

Reply:
Well, that is weird :)